invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/landing_page.less

```diff
@@ -153,3 +153,8 @@
          padding-top: 0px !important;
      }
  }
+
+ /* accordion cursor */
+ .ui.accordion .title {
+    cursor: auto !important;
```

Comment:
Out of curiosity why is this needed?

Reply:
marked area in the screenshot is the area where the cursor was a pointer.
![pointer](https://user-images.githubusercontent.com/44528277/109119197-a71b5580-7744-11eb-9f75-09a696265560.png)